openassessment/assessment/test/test_peer.py

```diff
@@ -1565,6 +1565,91 @@ def test_create_assessment_invalid_rubric_error(self):
                 REQUIRED_GRADED_BY
             )
 
+    def test_flexible_peer_grade_averaging(self):
```

Comment:
Why not use `ddt` instead of iterating over a test list here?

Reply:
Sorry, it didn't come into my mind. Updated.